appveyor.yml

```diff
@@ -1,15 +1,18 @@
-clone_folder: C:\gopath\src\github.com\just-install\just-install
 shallow_clone: true
 skip_branch_with_pr: true
 test: off
 environment:
-    GOPATH: C:\gopath
-    PATH: C:\gopath\bin;%PROGRAMFILES(x86)%\WiX Toolset v3.11\bin;%PATH%
+    PATH: C:\Go\bin;%PROGRAMFILES(x86)%\WiX Toolset v3.11\bin;%PATH%
```

Comment:
I'm not completely familiar with AppVeyor's default environment variables, but wouldn't it be better to use `%GOPATH%` instead of hardcoding `C:\Go`?

Reply:
AppVeyor stuff is meant to be temporary, all of this will be gone once AppVeyor rolls out new images.